src/user/user.manager.ts

```diff
@@ -26,12 +18,8 @@ export class UserManager {
         return UserRepository.getById(id);
     }
 
-    static getOne(userFilter: Partial<IUser>) {
-        return UserRepository.getOne(userFilter);
-    }
-
-    static getMany(userFilter: Partial<IUser>) {
-        return UserRepository.getMany(userFilter);
+    static getMany(userFilter: Partial<IUser>, startIndex: number, endIndex: number, sortOrder: string, sortBy: string) {
```

Comment:
`startIndex`, `endIndex`, `sortOrder` and `sortBy`should be optional fields in this layer as the controller might not pass all those properties and the repository as default value for undefined values

Reply:
Fixed.